README.md

```diff
@@ -26,7 +26,7 @@ The OpenTrials API service.
 2. Run `npm install`;
 3. Run `npm run migrate`;
 4. (Optional) If you want, you can add some seed data using `npm run seed`;
-5. Run `npm run reindex`;
+5. (If you ran 4) Run `npm run reindex`;
```

Comment:
I suspect that, if the user haven't added seed data, she'll have data in her local DB (e.g. loaded from a dump), so would need to run reindex anyway. Considering this, I prefer to keep step `5` as is.